Specification/BrAPI-Schema/BrAPI-Phenotyping/Scale.json

```diff
@@ -143,17 +137,17 @@
                 }
             },
             "required": [
-                "validValuesDbId"
+                "validValueDbId"
             ],
             "title": "ValidValues",
             "type": "object",
             "brapi-metadata": {
                 "primaryModel": false
             }
         },
-        "Categories": {
+        "Category": {
             "properties": {
-                "categoriesDbId": {
```

Comment:
There should be no DbId, for category, since it should be Value Type, and not an Entity.